js/setup.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const modal = document.querySelector(`.setup`);
+const template = document.querySelector(`#similar-wizard-template`).content.querySelector(`.setup-similar-item`);
+const modalElements = {
+  wizards: {
+    list: modal.querySelector(`.setup-similar-list`),
+    container: modal.querySelector(`.setup-similar`),
+    closeButton: modal.querySelector(`.setup-close`)
+  }
+};
+const names = [
+  `Иван`,
+  `Хуан Себастьян`,
+  `Мария`,
+  `Кристоф`,
+  `Виктор`,
+  `Юлия`,
+  `Люпита`,
+  `Вашингтон`
+];
+const surnames = [
+  `да Марья`,
+  `Верон`,
+  `Мирабелла`,
+  `Вальц`,
+  `Онопко`,
+  `Топольницкая`,
+  `Нионго`,
+  `Ирвинг`
+];
+const coatColors = [
+  `rgb(101, 137, 164)`,
+  `rgb(241, 43, 107)`,
+  `rgb(146, 100, 161)`,
+  `rgb(56, 159, 117)`,
+  `rgb(215, 210, 55)`,
+  `rgb(0, 0, 0)`
+];
+const eyesColors = [
+  `black`,
+  `red`,
+  `blue`,
+  `yellow`,
+  `green`
+];
+
+function getRandomElem(array) {
+  return array[window.getRandomInt(0, array.length - 1)];
+}
+
+function generateWizard() {
```

Comment:
```suggestion
function generateWizardDataObj() {
```
Ты тут генеришь объект с данными, следует это явно указать. Может показаться избыточным, но на больших проектах многословность в названии переменных - хорошая практика.